### PR TITLE
Two arguments to arc must be boolean.

### PR DESCRIPTION
**Pull request by mmachenry**
I've noticed that using the same "Show a => a -> a -> a" mechanism used on the rest of the functions in this module does not work for arcs. In the event that 'a' is an integer, it works fine. But as soon as you need to use a float, the two flag arguments become "0.0" or "1.0" and that is not valid SVG. I'm changing them to require booleans.

### CODE REVIEW

File: src/Text/Blaze/Svg/Internal.hs [hunk 1]
```diff
@@ -194,8 +194,8 @@ aa
   => a -- ^ Radius in the x-direction
   -> a -- ^ Radius in the y-direction
   -> a -- ^ The rotation of the arc's x-axis compared to the normal x-axis
-  -> a -- ^ Draw the smaller or bigger arc satisfying the start point
-  -> a -- ^ To mirror or not
+  -> Bool -- ^ Draw the smaller or bigger arc satisfying the start point
```
Comment (byorgey) [inline src/Text/Blaze/Svg/Internal.hs]: Could you clarify in the documentation which of (True, False) corresponds to (smaller, bigger)?

**Comment by byorgey** [comment]
Thanks, makes sense!  It might be nice to clarify the documentation a tiny bit as I noted in a comment above.

**Comment by mmachenry** [comment]
Good idea, I clarified the documentation.

**Comment by byorgey** [comment]
Uploaded as `0.3.6`.

**Pull request merged**